mobileClient/angularMobileApp/src/app/login/login.component.ts

```diff
@@ -0,0 +1,84 @@
+/*
```

Comment:
this file and many other files here seem to be an alternative window manager, as opposed to the two we currently have in zlux-app-manager/virtual-desktop/src/app/window-manager ... why dont we move the code there as a third window manager?